CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - **breaking change** Plugin measurements aggregated into a single measurement.
 - **breaking change** `jolokia` plugin: must use global tag/drop/pass parameters
 for configuration.
+- **breaking change** `postgresql` plugin: by default, converts both forms of address
```

Comment:
It's definitely not breaking in Telegraf proper; but it could certainly be breaking for tooling built around the data in influx.

but with the field aggregation its a kinda moot point vs 0.2